### PR TITLE
code style - mostly removing underscores from variable names

### PR DESCRIPTION
**Pull request by LeftofZen**

### CODE REVIEW

File: src/OpenLoco/SceneManager.cpp [hunk 1]
```diff
@@ -6,24 +6,24 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco
 {
-    loco_global<uint16_t, 0x00508F12> _screen_age;
+    loco_global<uint16_t, 0x00508F12> _screenAge;
     loco_global<uint16_t, 0x00508F14> _screenFlags;
-    loco_global<uint8_t, 0x00508F17> paused_state;
+    loco_global<uint8_t, 0x00508F17> pausedState;
```
Comment (duncanspumpkin) [inline src/OpenLoco/SceneManager.cpp:11]: Prefix with _

File: src/OpenLoco/Windows/BuildVehicle.cpp [hunk 2]
```diff
@@ -30,7 +30,7 @@ using namespace OpenLoco::Interop;
 
 namespace OpenLoco::Ui::Windows::BuildVehicle
 {
-    static const Ui::Size window_size = { 380, 233 };
+    static const Ui::Size windowSize = { 380, 233 };
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/BuildVehicle.cpp:33]: Should be k prefix and constexpr

File: src/OpenLoco/Windows/IndustryList.cpp [hunk 3]
```diff
@@ -578,7 +578,7 @@ namespace OpenLoco::Ui::Windows::IndustryList
     namespace NewIndustries
     {
 
-        static const Ui::Size window_size = { 578, 172 };
+        static const Ui::Size windowSize = { 578, 172 };
```
Comment (duncanspumpkin) [inline src/OpenLoco/Windows/IndustryList.cpp:581]: Same here. Should probably just check all the const items

**Review by duncanspumpkin — APPROVED** [review]
Looks sensible. Few further tweeks

File: src/OpenLoco/Windows/IndustryList.cpp (hunk 3 above)
Comment (LeftofZen) [reply src/OpenLoco/Windows/IndustryList.cpp:581]: I did this refactor and there were thousands of renames and LoC changed, and this PR is already too big. I'll do it in a separate PR just for windowSize?

Comment (duncanspumpkin) [reply src/OpenLoco/Windows/IndustryList.cpp:581]: Kk sounds good

File: src/OpenLoco/Interop/Hooks.cpp [hunk 4]
```diff
@@ -89,7 +89,7 @@ struct PaletteEntry
 };
 #pragma pack(pop)
 using SetPaletteFunc = void (*)(const PaletteEntry* palette, int32_t index, int32_t count);
-static Interop::loco_global<SetPaletteFunc, 0x0052524C> set_palette_callback;
+static Interop::loco_global<SetPaletteFunc, 0x0052524C> _set_palette_callback;
```
[the remark this answers is not in the document]
Comment (LeftofZen) [reply src/OpenLoco/Interop/Hooks.cpp:92]: its definitely used, in the conditional compilation blocks

**Pull request merged**